release/ray_release/exception.py

```diff
@@ -129,8 +129,8 @@ class TestCommandError(CommandError):
     exit_code = ExitCode.COMMAND_ERROR
 
 
-class ResultsError(CommandError):
-    pass
+class FetchResultError(ReleaseTestError):
```

Comment:
Kai, I don't think this should be CommandError. Let me know if it's ok with you.